apps.toml

```diff
@@ -591,6 +591,14 @@ state = "working"
 subtags = [ "network" ]
 url = "https://github.com/YunoHost-Apps/dex_ynh"
 
+[diacamma]
+category = "productivity_and_management"
+potential_alternative_to = ["Ciel Associations", "HelloAsso", "AssoConnect"]
+antifeatures = []
+state = "inprogress"
+subtags = ["accounting","business_and_ngos"]
+url = "https://github.com/Diacamma2/diacamma_ynh"
```

Comment:
It would be beneficial to transfer this repo to YunoHost-Apps org to allow automated testing.

Reply:
Quickly scanning through the contents of https://github.com/YunoHost-Apps/diacamma_ynh/tree/main I can tell this app could use some more work.